quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,34 @@
     },
```

Comment:
<p>Please add the new task types.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>